pkg/dispatcher/dispatcher.go

```diff
@@ -51,15 +51,15 @@ func NewDispatcher() Dispatcher {
 // Southbound listeners are only sent the events that matter to them
 // All events.Events are sent to northbound listeners
 func (d *Dispatcher) Listen(changeChannel <-chan events.ConfigEvent) {
-	log.Println("Event listener initialized")
+	log.Info("Event listener initialized")
 
 	for configEvent := range changeChannel {
-		log.Println("Listener: Event", configEvent)
+		log.Info("Listener: Event", configEvent)
 		deviceChan, ok := d.deviceListeners[topocache.ID(events.Event(configEvent).Subject())]
 		if ok {
-			log.Println("Device Simulators must be active")
+			log.Info("Device not connected - config event discarded", configEvent)
```

Comment:
this probably ought to be at least a warning

Reply:
done